### ISSUE TITLE
anki deadline can't see decks 

### ISSUE THREAD

**Issue by shyliece25** [BSCrumpton/Deadline2#12]
I need to add a deadline to more than one deck. however, when I try to set a date for my decks I only see one of my main decks
<img width="886" alt="Screen Shot 2021-04-08 at 12 31 14 PM" src="https://user-images.githubusercontent.com/81989597/114063157-511eef00-9866-11eb-980e-1bc08f258046.png">
<img width="548" alt="Screen Shot 2021-04-08 at 12 31 31 PM" src="https://user-images.githubusercontent.com/81989597/114063194-5a0fc080-9866-11eb-90ff-9c17141a0f15.png">

**Comment by BSCrumpton** [comment]
hey @shyliece25 , I'm trying to figure out what would be your issue is currently. 
I think in general, you're only seeing the FON 11,12,13,14 deck because that's the only once you currently have New Cards available for. 

I'm not 100% sure of what the issue is with Custom Study Session deck; but I'd assume it's because that deck is a "meta" deck and isn't real.

**Comment by wondermoi** [comment]
Hi ! As far as I'm concerned, Deadline cannot see any of my decks... Is it necessary to have New Cards available for the addon to be active and see Decks ? I'd like to set a deadline for an exam to come without adding New Cards on purpose, would that be possible ?

![anki_deadline_decks](https://user-images.githubusercontent.com/29425194/147007173-68480c9b-65c8-4046-a0c7-d2e176b8488f.png)
![anki_deadline_no_decks](https://user-images.githubusercontent.com/29425194/147007177-5d4c6d7a-d8b9-435f-b83d-e3b69b5d5942.png)

**Comment by Pelton** [comment]
I just reproduced the problem and that is the issue. To make a deck visible to me in Deadline's dropdown, I went to the list of decks, opened the deck that was missing, and added a new card [there are other ways to create a new card for deck; those were the steps I used in my case]. That done, Anki said I had a new card due today.  And at that point, the deck appeared in Deadline's drown. @BSCrumpton That may be something you wish to rectify. Here's my use case. I'm in a class that has a quiz each week on a particular chapter. Then, there's an exam that covers a subset of the classes chapters. At the end, there's a final. Every week, when I return from taking that week's quiz, I'm trying to reset the deadline so I review those cards for the upcoming exam. There are many paths to where I was, but, if you're wondering why someone would have no new cards due and also wish to change the deadline, that's mine.

**Comment by BSCrumpton** [comment]
heya @Pelton , I believe this is working as intended. The scheduling setup with new cards only works with ensuring you've "matured" X new cards per day, whereas once you have already matured the cards it's harder to figure out which cards you would actually still need to do. I guess the best path forward would be to either create another deck with the same cards (so that they're new), but that doesn't seem maintainable. probably just want to be able to reset the status of cards then, but doesn't fall under the scope currently of this add-on. Thanks for the comment!
